common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -686,17 +712,14 @@
             "off"
         ],
         "no-trailing-spaces": [
-            "error"
+            "off"
         ],
         "no-undef": [
             "off"
         ],
         "no-undef-init": [
             "error"
         ],
-        "no-underscore-dangle": [
-            "off"
```

Comment:
We use dangling underscores quite a bit, want to make sure this is still permitted.

Reply:
Where you see a rule that was previously off that is now removed completely, it means that the rule was always off by default, and the previous config explicitly turned it off unnecessarily.

I see how confusing this is. ☹️